sixtyfps_compiler/generator/rust.rs

```diff
@@ -910,12 +917,21 @@ fn generate_component(
             }
         ))
     } else if component.is_global() && component.visible_in_public_api() {
+        let aliases = component
+            .exported_global_names
+            .borrow()
+            .iter()
+            .flat_map(|name| (name != &component.root_element.borrow().id).then(|| ident(&name)))
```

Comment:
did you mean filter_map?

Reply:
or  `filter(...).map(...)`

Reply:
Indeed, that's easier to read than `flat_map` here.